src/dev/BQ76952.cpp

```diff
@@ -229,25 +341,60 @@ BQ76952::Status BQ76952::communicationStatus() {
 
 BQ76952::Status BQ76952::getCellVoltage(uint16_t cellVoltages[NUM_CELLS], uint32_t* sum) {
     Status status = Status::OK;
-    uint8_t cellVoltageReg = CELL_VOLTAGE_BASE_REG;
 
     uint32_t currentVoltage = 0;
     // Loop over all the cells and update the cooresponding voltage
     for (uint8_t i = 0; i < NUM_CELLS; i++) {
-        status = makeDirectRead(cellVoltageReg, &cellVoltages[i]);
+        status = makeDirectRead(CELL_REGS[i], &cellVoltages[i]);
         if (status != Status::OK) {
             return status;
         }
 
         currentVoltage += cellVoltages[i];
-
-        // Each cell register is 2 bytes off from each other
-        cellVoltageReg += 2;
     }
 
     *sum = currentVoltage;
 
     return BQ76952::Status::OK;
 }
 
+BQ76952::Status BQ76952::isBalancing(uint8_t targetCell, bool* balancing) {
+
+    uint32_t reg = 0;
+    RETURN_IF_ERR(makeRAMRead(0x83, &reg));
+
+    uint8_t targetLocation = CELL_BALANCE_MAPPING[targetCell - 1];
+
+    *balancing = reg >> targetLocation & 0x1;
+    return Status::OK;
+}
+
+BQ76952::Status BQ76952::setBalancing(uint8_t targetCell, uint8_t enable) {
+    // Read the current state, update the target cell, and write back out
+    // the data
+    uint32_t reg = 0;
+    RETURN_IF_ERR(makeRAMRead(0x83, &reg));
```

Comment:
You have a number of literals like this in the code. Can you abstract this into constants and document their purpose, so it's easier to follow what's going on?